.gitignore

```diff
@@ -12,8 +12,10 @@ build/
 .idea/
 build-eclipse/
 /tags
+out/
 
 locator
+server
```

Comment:
Is this needed?

Reply:
'Tis a vestige of a previous version of start.gfsh and will be removed forthwith.